### PR TITLE
wrong switch between secret and token

### PR DESCRIPTION
**Pull request by harshad16**
Found a minor error,
- The webhook secret was switched with access token; fixed it.

Label add for `approved`:
Might fix the #14

### CODE REVIEW

File: sesheta/webhook_processors/github_reviews.py [hunk 1]
```diff
@@ -59,3 +59,4 @@ def process_github_pull_request_review_submitted(pullrequest: dict, review: dict
     """Will handle with care."""
     if review["state"].startswith("approved"):
         _LOGGER.info("TODO set label 'approved' for {pullrequest['html_url']}")
```
Comment (goern) [inline sesheta/webhook_processors/github_reviews.py]: this aint a TODO anymore, syn?

**Review by goern — CHANGES_REQUESTED** [review]

File: sesheta/webhook_processors/github_reviews.py (hunk 1 above)
Comment (harshad16) [reply sesheta/webhook_processors/github_reviews.py]: updating :arrows_counterclockwise:

**Review by goern — APPROVED** [review]

**Comment by harshad16** [comment]
@goern  we would have to add the approved label here to get this merged.
I don't have access to do that, can you do that.

**Comment by goern** [comment]
hmm...

**Comment by harshad16** [comment]
@goern can you manually merge this.

**Comment by goern** [comment]
@harshad16 sorry, missed this somehow

**Pull request merged**